src/modules/micrortps_bridge/micrortps_client/microRTPS_client.h

```diff
@@ -77,7 +78,6 @@ struct options {
 	char ip[16] = IP;
 	uint16_t recv_port = DEFAULT_RECV_PORT;
 	uint16_t send_port = DEFAULT_SEND_PORT;
-	uint32_t update_time_ms = UPDATE_TIME_MS;
```

Comment:
`UPDATE_TIME_MS` can be removed as well.

Reply:
@bkueng done. Thanks!